lib/resolver/node-resolver.js

```diff
@@ -26,15 +26,30 @@ NodeResolver.prototype.resolveRule = function(pkg, ruleName) {
 
   pkg = this.normalizePkg(pkg);
 
+  // (1) try resolving rule via $PKG.rules[$NAME]
```

Comment:
Is this something we should test with the [configuration compiler](https://github.com/bpmn-io/bpmnlint/blob/master/lib/support/compile-config.js)?

Reply:
I forgot about that one. Will have another look!